### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by jportner**
Some small quallity-of-life improvements. See commit descriptions for details.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "snyk-github-issue-creator",
+    "name": "@elastic/snyk-github-issue-creator",
```
Comment (jportner) [inline package-lock.json:2]: I guess we forgot to change this when we moved the repo, but when I ran `npm install request` to update dependencies, it changed this in the lockfile too.

**Comment by jportner** [comment]
@pierre-ernst do you have some time to review this?

File: cli/utils.js [hunk 2]
```diff
@@ -68,14 +68,29 @@ const getGraph = (issue, prefix, showFullManifest) => {
...
+        if (val) {
+            acc.set(key, val + 1);
+        } else {
+            acc.set(key, 1);
+        }
+        return acc;
+    }, new Map());
+    return Array.from(pathsMap.entries())
+        .map(([k, v]) => (v === 1 ? k : k.replace('...', `... (x${v})`)))
         .join('\r\n');
```
Comment (jportner) [inline cli/utils.js:94]: When creating a new issue:

If there are lots of paths (more than 20) for a vulnerable transitive dependency, for example `z@1.0.0`:
```
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > w@1.0.0 > x@1.0.0 > y@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > w@1.0.0 > x@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > w@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > v@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > u@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > t@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > s@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > r@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > q@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > p@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > o@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > n@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > m@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > l@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > k@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > j@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > i@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > h@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > g@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > f@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > e@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > d@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > c@1.0.0 > z@1.0.0
project > a@1.0.0 > b@1.0.0 > z@1.0.0
project > a@1.0.0 > z@1.0.0
```

That will be reduced to one line:
```
project > a@1.0.0 > ... (x25) > z@1.0.0
```

We could bump the limit of 20 higher, I sort of picked that number arbitrarily. The reason for this change is that issues with too many transitive dependencies (think, hundreds) exceed GitHub's issue description character limit of 65536!

**Review by pierre-ernst — APPROVED** [review]
Sorry for the delay, thanks, looks great!

**Pull request merged**